compiler/cli_reporter.nim

```diff
@@ -2664,9 +2665,52 @@ proc reportShort*(conf: ConfigRef, r: ParserReport): string =
   reportBody(conf, r) & suffixShort(conf, r)
 
 
+proc genFeatureDesc[T: enum](t: typedesc[T]): string {.compileTime.} =
+  result = ""
+  for f in T:
+    if result.len > 0: result.add "|"
+    result.add $f
+
+const
```

Comment:
Admittedly, this feels weird in here, but I got nothing actionable

Reply:
In the future we would get rid of this compile-time string hardcoding and build CLI `--help` interface from some DSL/object, so for now it is sufficient, in the future it won't/ be needed.